Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "DataFrames"
```

Comment:
Do we need this change in this PR? I would focus on adding Tables.jl support only here.